### PR TITLE
Add country codes to institutions/ routes

### PR DESCRIPTION
**Pull request by jasonjia93**
Country codes are now required for the /institutions/ API endpoints. This updates the client library to properly take a country_codes field

### CODE REVIEW

**Review by cgfarmer4 — COMMENTED** [review]
LGTM. Ill let achen approve here so we can keep the release together.

File: test/test_institutions.rb [hunk 1]
```diff
@@ -18,43 +21,47 @@ def test_get_with_options
 
   def test_get_invalid_parameters
     assert_raises(Plaid::InvalidRequestError) do
-      client.institutions.get(count: BAD_STRING, offset: BAD_STRING)
+      client.institutions.get(count: BAD_STRING,
+                              offset: BAD_STRING,
+                              country_codes: ['US'])
```
Comment (otherchen) [inline test/test_institutions.rb:26]: I'm wondering if we should have a test for what happens if we use an empty country_codes array

**Review by otherchen — APPROVED** [review]
LGTM w/ a minor comment on adding a test to check for the correct error when an empty country_codes array is used in the API call
